.github/workflows/ci.yml

```diff
@@ -240,3 +240,59 @@ jobs:
         with:
           version: latest
           args: --all-targets --all-features
+
+  version_bump:
+    name: Ensure (MINOR) tag is used when making an API breaking change
+    # Change all of these steps to (MAJOR) after 1.0 release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        if: ${{ !contains(github.event.pull_request.title, '(MINOR)') }}
+      
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        if: ${{ !contains(github.event.pull_request.title, '(MINOR)') }}
+        with:
+          toolchain: nightly
```

Comment:
I know we are already dependent on nightly, but it would be good to avoid that complication if we can.
Will this nightly requirement change at some point?

Reply:
Thanks for the catch. Doesn't look like nightly is needed for this job, so I switched it to stable.